rskj-core/src/main/java/org/ethereum/vm/PrecompiledContracts.java

```diff
@@ -152,6 +152,9 @@ public PrecompiledContracts(RskSystemProperties config,
         this.remascConfig = new RemascConfigFactory(RemascContract.REMASC_CONFIG).createRemascConfig(config.netName());
     }
 
+    public boolean precompiledContactExists(RskAddress address) {
```

Comment:
why didn't you want to make use of `PrecompiledContracts#getContractForAddress()`? It respects activations, handles `null` address - everything that we care about

Reply:
I'll update that!